components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdpManager.java

```diff
@@ -24,7 +24,9 @@
 import org.wso2.carbon.identity.application.common.model.LocalRole;
 import org.wso2.carbon.identity.application.common.model.ProvisioningConnectorConfig;
 import org.wso2.carbon.identity.application.common.model.RoleMapping;
+import org.wso2.carbon.idp.mgt.object.IdpSearchResult;
```

Comment:
This package name is not correct. Looks like the package is renamed, but the change is not done here

Reply:
Already added: https://github.com/wso2/carbon-identity-framework/blob/48ce01208b0d609f00c188a06678e164d15d71c2/components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdpManager.java#L27